### PR TITLE
handle duplicate service creation

### PR DESCRIPTION
**Pull request by dmitrykruglov**

### CODE REVIEW

**Review by benoittoulme — DISMISSED** [review]
👍

File: provider/resource_service_test.go [hunk 1]
```diff
@@ -607,6 +607,6 @@ func assertServiceResourceJSONEq(t *testing.T, expected string, actual string, m
 			return kpis[i].(map[string]interface{})["_key"].(string) > kpis[j].(map[string]interface{})["_key"].(string)
 		})
 	}
-
+	delete(actualJSONAsInterface, "_key")
```
Comment (masha1997) [inline provider/resource_service_test.go:610]: I think it's better to add _key into actualJSONAsInterface. Pass some _key as additional param
and add this to the base:
```
// testOverridable
var GenerateUUID = func() (string, error) {
	return uuid.GenerateUUID()
}
```
and then override it in the test method

**Review by masha1997 — APPROVED** [review]

**Comment by masha1997** [comment]
Thank you for fixing this!

**Review by masha1997 — APPROVED** [review]
🥇

**Review by benoittoulme — APPROVED** [review]

**Pull request merged**